boskos/resources.yaml

```
@@ -46,6 +46,7 @@ resources:
   - istio-boskos-perf-01
   - istio-boskos-perf-02
   - istio-boskos-perf-05
+  - istio-boskos-perf-06
```

If we are adding two jobs, it may be better to add _two_ additional perf projects. There are times where we are at 100% utilization (ref: https://velodrome.istio.io/dashboard/db/boskos-dashboard?orgId=1&from=now-30d&to=now):

```suggestion
  - istio-boskos-perf-06
  - istio-boskos-perf-07
```

---

what's the 01, 02, 05 used for? Since previously we only have two perf job. Why do we have 3 perf clusters?

---

I guess, we removed one perf job, but did not remove the extra perf cluster? That's why I'm only adding one extra cluster. Or actually one of 01, 02 or 05 is not available or used?

---

Okay, I see. If there are only 4 jobs total, then 4 projects is sufficient.

---

yes, i think so.

---

Looking at the [code](https://github.com/istio/tools/blob/b7b1b8a3236ea95e733decb28df61cf4b1e95888/perf/benchmark/run_benchmark_job.sh#L37), anything running `perf/benchmark/run_benchmark_job.sh` and **not** overriding `RESOURCE_TYPE` is leasing a cluster from this pools. So, you are correct in that it is only **4**.